test/distributed_exec.jl

```diff
@@ -258,8 +258,8 @@ function test_regular_io_ser(ref::Base.Distributed.AbstractRemoteRef)
         v = getfield(ref2, fld)
         if isa(v, Number)
             @test v === zero(typeof(v))
-        elseif isa(v, Nullable)
-            @test v === Nullable{Any}()
+        elseif isa(v, Union{Some, Void})
+            @test v === nothing
         else
```

Comment:
This seems like a strange way to write it. Perhaps instead make the error branch an elseif and remove this `@test`?
```
elseif v !== nothing
```